src/framework/customModules/banner/hackableDynamic.js

```diff
@@ -36,6 +37,14 @@ var HackableDynamic = new Module.Class({
         textfilter: GObject.ParamSpec.string('textfilter', 'Text filter', '',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
             'normal'),
+        rotation: GObject.ParamSpec.uint('rotation', 'Rotation',
+            'Number of positions to advance each letter in the string',
+            GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY,
+            0, 25, 0),
+        decodefunc: GObject.ParamSpec.string('decodefunc', 'Decode Function',
+            'Source code of function to evaluate to decode the text',
+            GObject.ParamFlags.WRITABLE | GObject.ParamFlags.CONSTRUCT,
+            ''),
```

Comment:
I commented on Slack that this would feel like a good use case for an interface implemented by all the modules that rotate text, so that the boilerplate does not have to be repeated over and over.
But that can be added later in a follow up cleanup, so I won't block the PR on it.